requirements.txt

```diff
@@ -2,5 +2,7 @@ numpy
 scipy
 scikit-image
 boto
+pillow
```

Comment:
i'd rather not add this, and essentially make it optional by doing a check inside `frompng` and `topng`